# -*- coding: utf-8 -*-
class ValidationError(ValueError):
    """用于api_1_0中的errors.py模组中"""
    pass
